os/include/net/lwip/lwipopts.h

```diff
@@ -63,9 +63,16 @@
  * In this platform, DHCP client and server are supported through netutils
  * Instead of using lwIP DHCP, please use netutils dhcpc and dhcpd
  */
-#define LWIP_DHCPS                      0
-#define LWIP_DHCP                       1
-#define LWIP_NETIF_API                  1
+
+#ifdef CONFIG_NET_DHCPS
+#define LWIP_DHCPS CONFIG_NET_DHCPS
+#else
+#define LWIP_DHCPS 0
+#endif
+#ifdef CONFIG_NET_DHCP
+#define LWIP_DHCP CONFIG_NET_DHCP
```

Comment:
Thanks for the comment. That is defined in [opt.h](https://github.com/Samsung/TizenRT/blob/master/os/include/net/lwip/opt.h) line 836, when the client network stack options are determined.

Reply:
It looks much strange. You provide user selection to turn it on/off through Kconfig.
But regardless user selection, it will be determined by opt.h.
It can cause a big problem.
If you keep opt.h as current, let's not to add this config.

Reply:
I agree with you. Various network modules are tightly coupled with the dhcp structure of the network interface, so it will be better not to provide config of dhcp.  Then, dhcp will be automatically updated according to ipv4 setting. Thank you.